docs/start/getting-started/fragments/android/setup.md

```diff
@@ -56,10 +56,11 @@ Finally, update your AndroidManifest.xml with the following:
 In a terminal window, navigate to your project folder (the folder that typically contains your project level `build.gradle`) and run the following command (for this app, accepting all defaults is OK):
 
 ```bash
-$ cd ./YOUR_PROJECT_FOLDER
-$ amplify init        #accept defaults
+cd ./YOUR_PROJECT_FOLDER
+amplify init
 ```
-An `awsconfiguration.json` file will be created with your configuration and updated as features get added to your project by the Amplify CLI. The file is placed in the `./app/src/main/res/raw` directory of your Android Studio project and automatically used by the SDKs at runtime.
+
+Accept the default values. An `awsconfiguration.json` file will be created with your configuration and updated as features get added to your project by the Amplify CLI. The file is placed in the `./app/src/main/res/raw` directory of your Android Studio project and automatically used by the SDKs at runtime.
```

Comment:
user studies have shown that people don't read much outside of codeblocks. we should make these either bold or retain them in comments